atomic_reactor/utils/operator.py

```diff
@@ -276,11 +430,17 @@ def _container_pullspecs(self):
         ]
 
     def _annotation_pullspecs(self):
+        # Known sources of pullspecs in annotations
+
+        # TODO: Should this be a thing? Have known sources of pullspecs in annotations
```

Comment:
We want replace it and prevent using incorrect registry names in annotation

Reply:
I see, my question was more about whether we want non-heuristic annotations or just heuristic ones, and you've already answered that, thanks :)